Allow building extension when bundled with Slicer (#129)
vmtk/SlicerExtension-VMTK: CMakeLists.txt

find_package(Git REQUIRED)
mark_as_superbuild(GIT_EXECUTABLE)

> option(SlicerVMTK_enable_ExtraMarkups_and_StenosisMeasurement3D

Reviewer: This should be sufficient (if `ExtraMarkups` is disabled we could then disable all modules that depend on it).
```suggestion
option(SlicerVMTK_USE_ExtraMarkups
```

>   "This needs to be disabled when this extension is bundled with Slicer" ON)

Reviewer: ```suggestion
  "This needs to be disabled if ExtraMarkups Slicer extension is unavailable" ON)
```
